docs/src/main/paradox/file.md

```diff
@@ -68,6 +68,28 @@ Scala
 Java
 : @@snip (../../../../file/src/test/java/akka/stream/alpakka/file/javadsl/DirectoryChangesSourceTest.java) { #minimal-sample }
 
+### LogRotationSink
+
+The `LogRotationSink` will create and write to multiple files. There is two sink right now.
+
+The `functionBasedRotation` will call the given function every time when its 
+receive a new `ByteString` and if this function give new location the outputfile
+will change according to it. This is good for time based rotations 
+(like start a new file in every hour).
+
+The `sizeBasedRotation` will call the given function every time when the given size limit
+would be reached (so this generate lower then the limit filesizes). This is obiuosly good
+for size based rotations (like 'I want lower then 10Mb files').
```

Comment:
"The `sizeBasedRotation` works like the `functionBasedRotation` except that it only calls the provided path selection function when the file size plus the current bytes would go over a specific number of bytes. This is useful when you want to put an upper limit to the size of the written files.